package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@gusto/gusto-karma",
-  "version": "3.3.0",
+  "version": "4.3.0",
```

Comment:
can you keep the major version lockstep with the webpack version? maybe make this `3.4.0`?